testcases/open_posix_testsuite/bin/run-posix-option-group-test.sh

```diff
@@ -22,7 +22,7 @@ EOF
 
 run_option_group_tests()
 {
-	for test_script in $(find $1 -name run.sh | sort); do
+	for test_script in $(find $1 -name*.run-test | sort); do
```

Comment:
This should ideally be in single quotes to make sure that the asterisk is not expanded by accident i.e.
`-name '*.run-tests'`. Otherwise it looks good.

Also we slightly prefer patches being send to mailing list over pull requests.

Reply:
And the space after `-name` is mandatory.

Reply:
> This should ideally be in single quotes to make sure that the asterisk is not expanded by accident i.e. `-name '*.run-tests'`. Otherwise it looks good.
> 
> Also we slightly prefer patches being send to mailing list over pull requests.

The modified